plugins/exilepearl-paper/src/main/java/com/devotedmc/ExilePearl/command/CmdShowAllPearls.java

```diff
@@ -43,6 +44,7 @@ public class CmdShowAllPearls extends PearlCommand {
 	private static final DateFormat DATE_FORMAT = new SimpleDateFormat("dd MMM yyyy");
 	private static final Map<UUID, Long> COOLDOWNS = new HashMap<>();
 	private static final long COOLDOWN = 10_000; // 10 seconds
+	private boolean bannedPearlToggle = true;
```

Comment:
Just keep in mind this is a *global* toggle, so if someone hides banned players in their `/ep showall` window, then everyone else after that will also have banned players hidden. And when someone then unhides them, everyone after that will also see them.

Reply:
Huh, didn't realise. Is there a better way to do it then?

Reply:
You could do something similar to the `COOLDOWNS` map slightly above. Or you could make it a player setting and let people toggle it on or off without needing to run `/ep showall`, which might be preferred since it's a particularly taxing command to run. It'll also help with option persistence, as in, you don't need to open `/ep showall` and then re-open it with the toggle, doubling the strain on the server, to see what you want to see.

Reply:
Updated with the former idea